regression-tests/18-compile-arm-ports/Makefile

```diff
@@ -54,6 +54,8 @@ sensniff/ev-aducrf101mkxz \
 cfs-coffee/cc2538dk \
 cfs-coffee/openmote-cc2538 \
 cfs-coffee/zoul \
+fat/zoul:BOARD=remote-reva \
```

Comment:
On rev. A, Q4 is a MOSFET of a wrong type if the schematic symbol corresponds to the actual BoM (can you confirm?), so the SD card cannot even be powered properly. However, the card could be detected with the internal pull-down resistor of the MCU pad enabled.